CreeDictionary/API/models.py

```diff
@@ -1,19 +1,19 @@
 import logging
 import unicodedata
-from collections import defaultdict
-from typing import Dict, List, NamedTuple, Set, Tuple, Iterable
+from typing import List, NamedTuple, Set, Tuple, Iterable, NewType
 
 from attr import attrs
 from cree_sro_syllabics import syllabics2sro
 from django.db import models, transaction
 from django.db.models import Max, Q, QuerySet
+from sortedcontainers import SortedSet
 
-from constants import SimpleLC, SimpleLexicalCategory, POS
+from constants import SimpleLC, SimpleLexicalCategory, POS, Analysis
 from fuzzy_search import CreeFuzzySearcher
-from shared import descriptive_analyzer_foma, normative_generator
-from utils import fst_analysis_parser
+from shared import descriptive_analyzer_foma, normative_generator_foma
+from utils import fst_analysis_parser, get_modified_distance
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("django")
```

Comment:
`logging.getLogger(__name__)` is a common Python idiom with the logger library, and the [recommended way to do it](https://docs.python.org/3.8/library/logging.html#logger-objects) That way, log messages from just this module will be prefixed with "CreeDictionary.api.models.py".

I would change this back to the way it was originally.

```suggestion
logger = logging.getLogger(__name__)
```

Reply:
I know it's an idiom. I Just wanted to use the existing "django" handler and the logging handlers configured in `settings.py` so that the logger outputs both in console and the file in production. But I think that's achivable with proper configuration regarding logging propagation. I'll investigate later (or never, until I need investigate with logging again)